ui-components/src/forms/TextField.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import { useForm } from "react-hook-form"
+import { Field } from "./Field"
+
+export default {
+  title: "Forms/Text Field",
+  decorators: [(storyFn: any) => <div style={{ padding: "1rem" }}>{storyFn()}</div>],
+}
+
+export const TextFieldDefault = () => {
```

Comment:
What do you think about using just `Field` as a story name? I think this field is not just for text, but also for numbers, currency, etc. 🤔